trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > 10 ? name[0..10] : name
```

Comment:
- [ ] It should have a maximum of 10 characters. `name[0..10]` will give you 11 characters. Please tweak this to give the required result 👍